src/psij/executors/batch/test/qsub

```diff
@@ -16,7 +15,7 @@ try:
     args = [sys.executable, str(my_dir / 'qrun'), job_id, sys.argv[1]]
     proc = subprocess.Popen(args, stderr=f, stdout=subprocess.DEVNULL, text=True)
     logger.debug('job %s spawned', job_id)
-    time.sleep(0.05)
+    time.sleep(0.5)
     ec = proc.poll()
     if ec is not None:
```

Comment:
Is it not possible that the job could succeed within half a second? Could this maybe be `if ec != 0:`?

Reply:
> Is it not possible that the job could succeed within half a second

Yes, but it's not possible for qrun to exit in less than one second while having completed the job. There is a "queue time" of at least one second.

Reply:
Ooops, I missed the `setup.py` thing. Can you test now, please?

Reply:
It was fine. I lost track of it.